api/templates/foreign-travel.xml

```diff
@@ -15,6 +15,7 @@
     {{range $index, $item := .props.List.props.items}}
     {{with $Item := $item.Item}}
     <Travel ID="{{inc $index}}">
+      {{ if formType "SF86" }}
```

Comment:
ContactedByForeignMilitaryOrganization is present in the 85P – need to move the `if` statement down one element.

Reply:
nice catch!